app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -214,14 +225,41 @@ export default {
       }
     },
     deleteUnit(unit) {
-      this.form.ingredientMeasuresAttributes = this.form /* eslint-disable-line camelcase */
+      this.form.ingredientMeasuresAttributes = this.form
         .ingredientMeasuresAttributes.filter((originalUnit) => originalUnit !== unit);
       if (unit.id !== undefined) {
         this.measuresToDelete.push(unit.id);
       }
     },
     changeUnitName(unit, measure) {
       unit.name = measure;
+      this.autoAddUnit(unit);
+    },
+    autoAddUnit(currentUnit) {
+      const presentUnits = this.form.ingredientMeasuresAttributes.map((unit) => unit.name);
+      if (this.directConvertions[currentUnit.name] && currentUnit.quantity !== undefined) {
+        for (const unit of Object.keys(this.directConvertions[currentUnit.name])) {
```

Comment:
Muy personal, pero encuentro que queda más legible con .forEach que con un for

Reply:
Okay! Lo cambiaré